Sources/CollectionViewManager.swift

```diff
@@ -125,6 +126,23 @@ open class CollectionViewManager: NSObject {
 
         return sectionItem.cellItems[indexPath.row]
     }
+
+    /// Returns the reusable view item at the specified index path.
+    ///
+    /// - Parameter indexPath: The index path locating the item in the collection view.
+    /// - Returns: A reusable view item associated with reusable view of the collection, or nil if the view item
+    /// wasn't added to manager or indexPath is out of range.
+    open func reusableViewItem(for indexPath: IndexPath, and kind: String) -> CollectionViewReusableViewItem? {
```

Comment:
```suggestion
    open func reusableViewItem(for indexPath: IndexPath, kind: String) -> CollectionViewReusableViewItem? {
```
It's generally better to leave parameter name 'as is' for STL types, to avoid confusion regarding their meaning.